week-3/slideshow/style.css

```diff
@@ -1 +1,25 @@
 /** Write your CSS in here **/
+body{
+    display: flex;
+    flex-direction: column;
+align-items: center;
+background-color: black;}
+img {
+    height: 300px;
+}
+p{
+    color: #ea4c89;
+}
+button{
+background-color: #ea4c89;
+border-radius: 1rem;
+border: none;
+color: white;
+padding: 15px 32px;
+text-align: center;
+text-decoration: none;
```

Comment:
I appreciate the effort to make the buttons look good. They loom pretty good 👍

Reply:
🙂